app/Models/Auth.php

```diff
@@ -47,8 +47,8 @@ public static function init() {
 	 * @return boolean true if user can be connected, false else.
 	 */
 	private static function accessControl() {
-		$conf = Minz_Configuration::get('system');
-		$auth_type = $conf->auth_type;
+		FreshRSS_Context::$system_conf = Minz_Configuration::get('system');
```

Comment:
I will make another PR with a better refactoring of this part, but which requires a few global changes.

Reply:
https://github.com/FreshRSS/FreshRSS/pull/3070